src/styles/photoList.styl

```diff
@@ -0,0 +1,56 @@
+@require 'cozy-ui'
+
+:local
+
+    .pho-section
+        left-spaced(32px)
+
+    .pho-photo
+        position relative
+        display  inline-block
+        background-color grey-10
+        margin   0 1em 1em 0
+
+        a
+          vertical-align middle
+
+    .pho-photo-item
+        height       em(240px)
+        transition   transform .2s ease
+
+    .pho-photo-select
+        position absolute
+        z-index  2
+        opacity  0
+        height   3em
+        width    3em
+        padding  1em
+        cursor   pointer
+
+        &[data-input=checkbox] label::before
+            box-shadow 0 0 0 3px rgba(0,0,0,.3), inset 0 0 0 1.5em rgba(255,255,255,.8)
```

Comment:
Aren't the checkboxes supposed to be the same everywhere, whether it's in Files or Photos ? 
If so, just move your modification to Cozy-UI directly, if not, I'd like to know if it could be used in any other incoming app and if so ->  Cozy-UI if not it stays here.
Basically I think it should be in Cozy-UI for consistency. But I'd like to know what our mavelous @kelukelu has in mind.